benchmarks/tract-onnx-image-classification/setup.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+set -euxo pipefail
+
+MODEL_URL="https://www.github.com/onnx/models/raw/main/validated/vision/classification/mobilenet/model/mobilenetv2-7.onnx?download=:"
+MODEL_FILE="assets/mobilenetv2-7.onnx"
+
+if [ ! -f $MODEL_FILE ]; then
+	echo "Downloading model to $MODEL_FILE"
+
+	if which wget >/dev/null ; then
+	wget $MODEL_URL -O $MODEL_FILE
```

Comment:
An assets directory needs to be created first.

Reply:
Oops, thanks I'll fix that.